[2024-06-28T13:58:25.307+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-28T13:58:25.390+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_sample_dag.task_from_script1 scheduled__2023-08-16T02:35:00+00:00 [queued]>
[2024-06-28T13:58:25.409+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_sample_dag.task_from_script1 scheduled__2023-08-16T02:35:00+00:00 [queued]>
[2024-06-28T13:58:25.414+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-28T13:58:25.470+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): task_from_script1> on 2023-08-16 02:35:00+00:00
[2024-06-28T13:58:25.507+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1204) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-28T13:58:25.511+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'my_sample_dag', 'task_from_script1', 'scheduled__2023-08-16T02:35:00+00:00', '--job-id', '1799', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpzfq_ospr']
[2024-06-28T13:58:25.517+0000] {standard_task_runner.py:91} INFO - Job 1799: Subtask task_from_script1
[2024-06-28T13:58:25.512+0000] {standard_task_runner.py:63} INFO - Started process 1270 to run task
[2024-06-28T13:58:25.733+0000] {task_command.py:426} INFO - Running <TaskInstance: my_sample_dag.task_from_script1 scheduled__2023-08-16T02:35:00+00:00 [running]> on host 81089ac3a69d
[2024-06-28T13:58:26.010+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_sample_dag' AIRFLOW_CTX_TASK_ID='task_from_script1' AIRFLOW_CTX_EXECUTION_DATE='2023-08-16T02:35:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-16T02:35:00+00:00'
[2024-06-28T13:58:26.014+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-28T13:58:26.016+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-28T13:58:26.017+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/script1.py", line 10, in function_from_script1
    response = requests.get(url)
                            ^^^
NameError: name 'url' is not defined
[2024-06-28T13:58:26.036+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=my_sample_dag, task_id=task_from_script1, run_id=scheduled__2023-08-16T02:35:00+00:00, execution_date=20230816T023500, start_date=20240628T135825, end_date=20240628T135826
[2024-06-28T13:58:26.061+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 1799 for task task_from_script1 (name 'url' is not defined; 1270)
[2024-06-28T13:58:26.104+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-28T13:58:26.127+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
